patientsearch.env.default

```diff
@@ -1,6 +1,7 @@
 # docker-compose default environment file
-# copy to dashboard.env and modify as necessary
+# copy to patientsearch.env and modify as necessary
 
+EXTERNAL_SEARCH_URL=https://cosri-pdmp.cirg.washington.edu/v/r2/fhir/
```

Comment:
```suggestion
FHIR_BASE_URL=https://cosri-pdmp.cirg.washington.edu/v/r2/fhir/
```
Not strictly specific to search, could extend to call other FHIR APIs

Reply:
Correct, but we need to somehow clarify "external" (that is PDMP in this case) versus the fEMR internal HAPI store, which FHIR_BASE_URL sounds likely to represent.